### PR TITLE
Why did I join source{d}? Francesc

### PR DESCRIPTION
**Pull request by campoy**
required by https://github.com/src-d/blog/issues/175

It can be reviewed in staging at https://blog-staging.srcd.run/post/why-i-joined-sourced-francesc/

### CODE REVIEW

File: content/post/why-i-joined-sourced-francesc.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+author: campoy
+date: 2018-01-15
+title: "Why did I join source{d}? - Francesc Campoy"
+draft: true
+description: "The first post of a series on why multiple employees joined source{d}. This one is by Francesc Campoy."
+categories: ["why-i-joined"]
```
Comment (dpordomingo) [inline content/post/why-i-joined-sourced-francesc.md]: Should this post appear in our landing, under the "culture" category?
If so, it should be also categorized as `culture`

File: content/post/why-i-joined-sourced-francesc.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+---
+author: campoy
+date: 2018-01-15
+title: "Why did I join source{d}? - Francesc Campoy"
+draft: true
+description: "The first post of a series on why multiple employees joined source{d}. This one is by Francesc Campoy."
```
Comment (dpordomingo) [inline content/post/why-i-joined-sourced-francesc.md]: The article lacks image, what breaks our listings. What if we ask our designer for a header to this kind of posts?

File: content/post/why-i-joined-sourced-francesc.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+all of the Go code available on GitHub.
+
+Later on, this kind of analysis started to become a requirement to justify
+additions to Go's standard library.
+
+For instance, the `time.Until` function was added to Go with this
+[proposal](https://github.com/golang/go/issues/14595) after an analysis of
+how many times we could find an equivalent piece of code on GitHub.
+
+![screenshot from the proposal](img/hello-sourced/issue.png)
```
Comment (dpordomingo) [inline content/post/why-i-joined-sourced-francesc.md]: missing image

File: content/post/why-i-joined-sourced-francesc.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+What's my favorite part of the company? The incredibly incredibly talented team tackling
+problems at different layers:
+
+- retrieval: let's fetch all of the code in the world.
+- engine: creating a declarative API to access all of that code.
+- ML: building and training models that can power complex applications.
+- applications: finding the use cases that allows us to prove the power of the platform, while providing as much benefit to the community as possible.
+
+![we want you](img/hello-sourced/wewantyou.jpg)
```
Comment (dpordomingo) [inline content/post/why-i-joined-sourced-francesc.md]: missing image

File: content/post/why-i-joined-sourced-francesc.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+### So what's coming?
+
+As VP of Developer Relations my job is to strategize how source{d} can empower developers
+all around the world to write better code by:
+
+- writing new models that will power amazing tools,
+- building tools powered by ML on code, and of course
+- using those developer tools to analyze and improve source code.
+
+source{d} is building a platform by *developers* where *developers* can build tools for *developers* [[1]](https://www.youtube.com/watch?v=KMU0tzLwhbE).
```
Comment (dpordomingo) [inline content/post/why-i-joined-sourced-francesc.md]: what if we embed the video? would it be too funny? :P

**Comment by dpordomingo** [comment]
About links: could we consider using [descriptive links](http://oregonstate.edu/accessibility/descriptivelinks), as [it was suggested by Ricardo](https://github.com/src-d/guide/pull/84/files#r160802043)? or does it only affects our documentation?
If we should, tell me and I'll suggest a couple of changes more.

File: content/post/why-i-joined-sourced-francesc.md (hunk 2 above)
Comment (campoy) [reply content/post/why-i-joined-sourced-francesc.md]: I copy pasted and totally forgot about the images, good catch!
Adding them now.

File: content/post/why-i-joined-sourced-francesc.md (hunk 1 above)
Comment (campoy) [reply content/post/why-i-joined-sourced-francesc.md]: done

File: content/post/why-i-joined-sourced-francesc.md (hunk 3 above)
Comment (campoy) [reply content/post/why-i-joined-sourced-francesc.md]: fixed

File: content/post/why-i-joined-sourced-francesc.md (hunk 4 above)
Comment (campoy) [reply content/post/why-i-joined-sourced-francesc.md]: fixed

File: content/post/why-i-joined-sourced-francesc.md (hunk 5 above)
Comment (campoy) [reply content/post/why-i-joined-sourced-francesc.md]: I think it's better to keep it as a surprise, people click without knowing what they're gonna get and...

steve ballmer

**Comment by campoy** [comment]
Feedback applied, thanks @dpordomingo

**Comment by dpordomingo** [comment]
Thank you for summarizing the links!

The only thing I miss is the post image (otherwise it will break our listings)

**Comment by campoy** [comment]
Maybe @ricardobaeta can suggest an image that we can use for all the "why I joined source{d}" blog posts to come?

**Comment by ricardobaeta** [comment]
Sure @campoy! Let me read the article for insights on a rationale. Thanks for the mention :)

**Comment by ricardobaeta** [comment]
@campoy I think I get the picture. I'll work on some approaches tomorrow - because we have the 15th as a deadline :)

**Comment by ricardobaeta** [comment]
Hi Francesc! I’m sorry but today was underwater. I will work on the image Monday Morning. What would be the best time for you to publish the post?

**Comment by campoy** [comment]
We can also publish this a bit later, I'm currently working on the schedule for the blog
I wrote that date as a date I'd like to have, no strong deadline

**Comment by ricardobaeta** [comment]
@campoy https://src-d.slack.com/archives/C7TG7TE7K/p1516380708000355

**Comment by campoy** [comment]
Cool! I love it. Could you send a PR adding the image somewhere in the blog so we can use it in the future?

**Comment by ricardobaeta** [comment]
@dpordomingo I've created this image to address this need. The only remark is that I found it impossible to work on a canvas of 90px height like we have now. So the approved image has 157px height. Is it possible to adapt the code of the blog to, as from now, display images with this size? I would say that it's an improvement. What do you think?

why-did-i-join-sourced-template-image

> the image is @2x for retina

 @campoy Here's the mockup with the template image. Let's just wait for David's feedback to move forward to the PR.

why-did-i-join-sourced-final

**Comment by campoy** [comment]
Hey @ricardobaeta please avoid links to private resources (such as a private slack channel) from public repos :)

**Comment by campoy** [comment]
What did just happen here? I pushed a new version to my fork and this happens? 😕 
I'll create a new PR

**Pull request closed without merging**